jobs/build/merge_ocp/Jenkinsfile

```diff
@@ -21,12 +21,16 @@ node {
                         $class: 'hudson.model.StringParameterDefinition',
                         defaultValue: commonlib.ocpMergeVersions.join(',')
                     ],
-                    [
-                        name: 'COMMIT_DEPTH',
-                        description: 'How deep to clone to ensure merges have a common base; 0 for full depth',
-                        $class: 'hudson.model.StringParameterDefinition',
-                        defaultValue: "0"
-                    ],
+                    booleanParam(
+                        name: 'CLEAN_CLONE',
+                        defaultValue: false,
+                        description: 'Force all git repos to be re-pulled'
+                    ),
+                    booleanParam(
+                        name: 'SCHEDULE_INCREMENTAL',
+                        defaultValue: true,
+                        description: 'If changes are detected, schedule an incremental build (4.x only)'
+                    ),
```

Comment:
Something I'd add here would be like we have in a few other jobs
https://github.com/openshift/aos-cd-jobs/blob/master/jobs/build/ocp4/Jenkinsfile#L76
`commonlib.suppressEmailParam(),`
Won't send emails if it's not in the main `aos-cd-jobs` place. Not required, just a suggestion.